Makefile

```diff
@@ -0,0 +1,37 @@
+ENVIRONMENT ?= betacloud
+
+VERSION_CEPH ?= pacific
+VERSION_MANAGER ?= latest
+VERSION_OPENSTACK ?= yoga
+
+clean:
+	cd terraform && make ENVIRONMENT=$(ENVIRONMENT) clean
```

Comment:
I'd very much prefer to use `make -C terraform ...` instead of this `cd && make` construct